storybook/preview.js

```diff
@@ -36,15 +38,26 @@ import './polyfill-globals';
 import { setUsingCache } from 'googlesitekit-api';
 import { resetGlobals } from './utils/resetGlobals';
 import { bootstrapFetchMocks } from './fetch-mocks';
-import { WithTestRegistry } from '../tests/js/utils';
 import { enabledFeatures } from '../assets/js/features';
 import { Cell, Grid, Row } from '../assets/js/material-components';
-import { setEnabledFeatures } from '../tests/js/test-utils';
+import {
+	createTestRegistry,
+	provideUserInfo,
+	setEnabledFeatures,
+} from '../tests/js/test-utils';
+import InViewProvider from '../assets/js/components/InViewProvider';
+import { RegistryProvider } from '../assets/js/googlesitekit-data';
```

Comment:
`googlesitekit-data` should always be imported from directly via its alias (see `googlesitekit-api` above).